src/webui/static/app/dvr.js

```diff
@@ -490,8 +498,8 @@ tvheadend.dvr_failed = function(panel, index) {
         url: 'api/dvr/entry',
         gridURL: 'api/dvr/entry/grid_failed',
         readonly: true,
-        titleS: _('Failed Recording'),
-        titleP: _('Failed Recordings'),
+        titleS: _('Failed recording'),
+        titleP: _('Failed recordings'),
         iconCls: 'exclamation',
```

Comment:
Are they now "failed" or "missed" (or similar)? Disabled recordings appear here as well, but they're not strictly failures, are they...